lms/djangoapps/teams/tests/test_views.py

```diff
@@ -296,6 +333,10 @@ def test_page(self):
         self.assertIsNone(result['next'])
         self.assertIsNotNone(result['previous'])
 
+    def test_expand_user(self):
+        result = self.get_teams_list(200, {'expand': 'user', 'topic_id': 'renewable'})
+        self.assertIn('email', result['results'][0]['membership'][0]['user'])
```

Comment:
I would suggest a helper method that verifies an expanded user. I agree that it doesn't have to check all fields, but at least that hides what we are checking.